src/requests/utils.py

```diff
@@ -540,7 +541,7 @@ def get_encoding_from_headers(headers):
 
     content_type = headers.get("content-type")
 
-    if not content_type:
+    if content_type:
         return None
```

Comment:
**Suggestion:** The 'if' condition seems to be incorrect. It should be 'if not content_type:' instead of 'if content_type:'.
```suggestion
    if not content_type:
        return None
```